pages/htmlresources.md

```diff
@@ -216,12 +216,15 @@ Now delete your dummy resource by clicking `Delete`. Then click `Open` on you ht
 
 Finally, push your app to the nation by clicking `Add to Nation`.
 
+_Note:_
+Friendly reminder again to use Firefox when doing these steps or 'Add to Nation' may not appear in other browsers.
+
```

Comment:
these are the only 3 lines that were added. Try to avoid changing anything in the rest of the file to not confuse reviews into thinking there were actually a bunch of changes made.